boards/arm/rpi_pico/doc/index.rst

```diff
@@ -86,6 +86,12 @@ Flashing
 Using an SWD adapter
 --------------------
 
+You can Flash the rpi_pico with a SEGGER JLink debug probe with
```

Comment:
This is the 'Using an SWD adapter' section, and the JLink debug probe is not an SWD adapter.

End of this section, line
https://github.com/zephyrproject-rtos/zephyr/blame/52fe06495dc2d9bfa394353778c3f211441a65fb/boards/arm/rpi_pico/doc/index.rst#L134-L139
 is mentioned about how to use JLink.
I think it should be replaced with a new section (I think it is good to be titled 'Using J-Link adapter')
and put this command line into it.

Reply:
I would call the JLink a SWD (and JTAG, SPI, ...) adapter. I could split it up into `Using jlink` and `Using openocd`

Reply:
`I would call the JLink a SWD (and JTAG, SPI, ...) adapter.`
Sorry, This is correct.

` I could split it up into Using jlink and Using openocd`
👍 
It is more kindly for users. Thank you for suggestion.